ecommerce/views.py

```diff
@@ -128,6 +128,16 @@ def get_serializer_context(self):
         """
         return {"has_ordered_versions": True}
 
+    def list(self, request, *args, **kwargs):
+        """
+        Sort the default response if indicated in query string
+        """
+        response = super().list(request, *args, **kwargs)
+        sort = request.GET.get("sort")
+        if response.status_code == status.HTTP_200_OK and sort == "title":
```

Comment:
Nitpick: we can just use `response.ok` instead of `response.status_code == status.HTTP_200_OK`

Reply:
Getting an error: `Response object has no attribute ok`.